Sources/Shared/Data Manager/Proxies/Social/SocialSpaceProxy.h

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (C) 2003-2015 eXo Platform SAS.
+//
+// This is free software; you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as
+// published by the Free Software Foundation; either version 3 of
+// the License, or (at your option) any later version.
+//
+// This software is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this software; if not, write to the Free
+// Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+// 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+//
+
+
+#import "SocialProxy.h"
+#import "SocialSpace.h"
+@interface SocialSpaceProxy : SocialProxy {
+    NSArray*                    _mySpaces;
+}
+@property (nonatomic,retain) NSArray * mySpaces;
+
+-(void) getMySocialSpaces;
+
+-(void) getIdentifyOfSpace:(SocialSpace *) space;
+
```

Comment:
Please add some [Xcode doc](https://developer.apple.com/library/mac/documentation/DeveloperTools/Conceptual/HeaderDoc/tags/tags.html) to explain what these methods do. In particular, explain that both of them, although they get different responses from the REST services, map to the same array `mySpaces`. Explain that in the case of `getMySocialSpaces` it's a "normal" array of `SocialSpace` that is returned, but in the case of `getIdentifyOfSpace` it's an array with one `SocialSpace` element that contains only the `spaceId` attribute.